src/UCTNode.h

```diff
@@ -112,7 +113,7 @@ class UCTNode {
     SMP::Mutex m_nodemutex;
 
     // Tree data
-    std::atomic<bool> m_has_children{false};
+    std::atomic<float> m_min_psa_ratio_children{2.0f};
```

Comment:
Does changing this from bool to float blow up the node size? If so, you might be able to extend m_status to have a PARTIALLY_EXPANDED state and use that instead.